src/App.js

```diff
@@ -1,8 +1,16 @@
 import React from 'react';
 import './App.scss';
+import PropTypes from 'prop-types';
 
-const App = () => (
-  <h1>React sum</h1>
+const App = ({ a, b }) => (
+  <h1>
+    {`Sum of ${a} and ${b} is ${a + b}`}
+  </h1>
```

Comment:
Inside `App` component there should be three instances of `Sum` components (that you need to create).